### PR TITLE
FHG-2371 Fixed prepending of web site

### PR DESCRIPTION
**Pull request by MartinBelton-gov**
FHG-2371 Fixed prepending web site

### CODE REVIEW

File: tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs [hunk 1]
```diff
@@ -44,7 +56,7 @@ public async Task ThenOnGetAsync_LocalOfferDetailWithReferralNotEnabled()
         localOfferDetailModel.PageContext.HttpContext = httpContext;
 
         //Act 
-        var result = await localOfferDetailModel.OnGetAsync("NewId", serviceDto.Id) as PageResult;
+        var result = await localOfferDetailModel.OnGetAsync("NewId", (serviceDto != null) ? serviceDto.Id : string.Empty) as PageResult;
 
         //Assert
```
Comment (hardik-desai-kcl) [inline tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs:61]: could you please add an assert Website.should.be.equalto(url)?

File: tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs [hunk 2]
```diff
@@ -19,8 +19,11 @@ namespace FamilyHubs.ReferralUi.UnitTests.Pages.ProfessionalReferral;
 
 public class WhenUsingLocalOfferDetail
 {
-    [Fact]
-    public async Task ThenOnGetAsync_LocalOfferDetailWithReferralNotEnabled()
+    [Theory]
+    [InlineData(default!)]
+    [InlineData("url")]
+    [InlineData("https://wwww.google.com")]
```
Comment (hardik-desai-kcl) [inline tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs:25]: Please could you add test for http://google.com

Comment (MartinBelton-gov) [reply tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs:25]: Added as requested

File: tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs (hunk 1 above)
Comment (MartinBelton-gov) [reply tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs:62]: Does not exactly work like that but changed anyway

**Review by hardik-desai-kcl — APPROVED** [review]

File: tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs [hunk 3]
```diff
@@ -44,11 +57,21 @@ public async Task ThenOnGetAsync_LocalOfferDetailWithReferralNotEnabled()
...
 
         //Assert
         result.Should().NotBeNull();
         result.Should().BeOfType<PageResult>();
+        if(url == null)
+        {
+            localOfferDetailModel.Website.Should().BeEquivalentTo("");
+        }
+        else if (url == "url")
+            localOfferDetailModel.Website.Should().BeEquivalentTo("https://url");
```
[the remark this answers is not in the document]
Comment (MartinBelton-gov) [reply tests/FamilyHubs.ReferralUi.UnitTests/Pages/ProfessionalReferral/WhenUsingLocalOfferDetail.cs]: If it's a doggy url then an empty string is returned

**Review by hardik-desai-kcl — APPROVED** [review]

**Pull request merged**